src/iter/iter.rs

```diff
@@ -1,17 +1,23 @@
 use super::NodeIter;
+use core::sync::atomic::Ordering;
 use crossbeam_epoch::Guard;
-use std::sync::atomic::Ordering;
 
 /// An iterator over a map's entries.
 ///
 /// See [`HashMap::iter`](crate::HashMap::iter) for details.
 #[derive(Debug)]
-pub struct Iter<'g, K, V> {
-    pub(crate) node_iter: NodeIter<'g, K, V>,
+pub struct Iter<'g, K, V, L>
```

Comment:
Maybe add a default `L` for all of the iterator types too?

Reply:
Bah, then we'd have to duplicate all of these too. One other option here is to define a default value on `no_std` that panics when used. It ain't great at all, but it would allow us to not duplicate all the structs.

Reply:
I suppose that's why `hashbrown` uses an uninhabited default on no-std.

Reply:
No, hashbrown uses ahash on no_std. It only uses an uninhabited default when compiled as a dependency for libstd (because the libstd build system is weird and makes dependencies difficult).

Reply:
Oh ok, I misunderstood that context, and hashbrown doesn't need a lock like this. An uninhabited default might still work though, or we could use `macro_rules!` to hide the duplication.